canarytokens/models.py

```diff
@@ -1308,7 +1308,7 @@ def get_additional_data_for_notification(self) -> Dict[str, Any]:
             Dict[str, str]: Key value pairs to include in webhook info.
         """
 
-        additional_info = json_safe_dict(
+        additional_data = json_safe_dict(
```

Comment:
why the rename here? makes the diff a little harder to read / see where the actual changes are?